sbin/verf-cli.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
nit: Here, and for other files - explicitly rename the script to `verifier-*.sh` ? (internal variables like `CELEBORN_VERF_CLI_JAVA_OPTS` is fine : the scripts get exposed to users).
We dont gain much by the shortening (and it is not a standard name from what I can tell)